syntax = "proto3";

import "google/protobuf/empty.proto";

service DataService {
    rpc EnviarParticion(NombreDataNode) returns (Particion) {}
    rpc RecibirParticion(Particion) returns (google.protobuf.Empty) {}
}

message NombreDataNode {
    string nombre = 1;
}

message Particion {
    bytes contenido = 1;
    string fileInfo = 2;
}